src/Microsoft.DncEng.SecretManager/Program.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Microsoft.DncEng.CommandLineLib;
+using Microsoft.DncEng.CommandLineLib.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.DncEng.SecretManager
+{
+    public class Program : DependencyInjectedConsoleApp
+    {
+        static async Task<int> Main(string[] args)
+        {
+            return await new Program().RunAsync(args);
```

Comment:
Does it need the "async" "return await"? Could it just return the Task itself?

Reply:
Yep, I will do that.